[2024-09-11T01:59:55.028+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-11T01:59:55.093+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.seccond_task scheduled__2024-08-20T00:00:00+00:00 [queued]>
[2024-09-11T01:59:55.114+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.seccond_task scheduled__2024-08-20T00:00:00+00:00 [queued]>
[2024-09-11T01:59:55.115+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-09-11T01:59:55.294+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): seccond_task> on 2024-08-20 00:00:00+00:00
[2024-09-11T01:59:55.312+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=12716) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T01:59:55.311+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'seccond_task', 'scheduled__2024-08-20T00:00:00+00:00', '--job-id', '3432', '--raw', '--subdir', 'DAGS_FOLDER/7. our_first_dag.py', '--cfg-path', '/tmp/tmpu8z_bdkd']
[2024-09-11T01:59:55.316+0000] {standard_task_runner.py:91} INFO - Job 3432: Subtask seccond_task
[2024-09-11T01:59:55.315+0000] {standard_task_runner.py:63} INFO - Started process 12737 to run task
[2024-09-11T01:59:55.419+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v4.seccond_task scheduled__2024-08-20T00:00:00+00:00 [running]> on host 0359316e7bd1
[2024-09-11T01:59:55.600+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='seccond_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-20T00:00:00+00:00'
[2024-09-11T01:59:55.601+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-11T01:59:55.621+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-11T01:59:55.622+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "hey, I am task2 and will be running after task 1"']
[2024-09-11T01:59:55.637+0000] {subprocess.py:86} INFO - Output:
[2024-09-11T01:59:55.641+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task 1
[2024-09-11T01:59:55.642+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-11T01:59:55.643+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-11T01:59:55.698+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=seccond_task, run_id=scheduled__2024-08-20T00:00:00+00:00, execution_date=20240820T000000, start_date=20240911T015955, end_date=20240911T015955
[2024-09-11T01:59:55.777+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-11T01:59:55.794+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
